packages/client/hmi-client/src/components/workflow/ops/calibrate-ensemble-ciemss/tera-calibrate-ensemble-node-ciemss.vue

```diff
@@ -63,7 +63,9 @@ const emit = defineEmits(['open-drilldown', 'update-state', 'append-output', 'ap
 const runResults = ref<RunResults>({});
 const csvAsset = shallowRef<CsvAsset | undefined>(undefined);
 
-const areInputsFilled = computed(() => props.node.inputs[0].value && props.node.inputs[1].value);
+const areInputsFilled = computed(
+	() => props.node.inputs[0].value && props.node.inputs[1].value && props.node.inputs[2].value
+);
```

Comment:
Note: This technically would not work, if somehow I ended up with
1. model-config
2. nothing after disconnection
3. dataset
4. model-config

Reply:
fair point I have updated to filter on modelConfigId's that have values assigned